lib/cucumber/core/platform.rb

```diff
@@ -5,12 +5,23 @@
 
 module Cucumber
   unless defined?(Cucumber::VERSION)
-    JRUBY         = defined?(JRUBY_VERSION)
-    IRONRUBY      = defined?(RUBY_ENGINE) && RUBY_ENGINE == "ironruby"
-    WINDOWS       = RbConfig::CONFIG['host_os'] =~ /mswin|mingw/
-    OS_X          = RbConfig::CONFIG['host_os'] =~ /darwin/
-    WINDOWS_MRI   = WINDOWS && !JRUBY && !IRONRUBY
-    RUBY_2_0      = RUBY_VERSION =~ /^2\.0/
-    RUBY_1_9      = RUBY_VERSION =~ /^1\.9/
+    JRUBY          = defined?(JRUBY_VERSION)
+    IRONRUBY       = defined?(RUBY_ENGINE) && RUBY_ENGINE == "ironruby"
+    WINDOWS        = RbConfig::CONFIG['host_os'] =~ /mswin|mingw/
+    OS_X           = RbConfig::CONFIG['host_os'] =~ /darwin/
+    WINDOWS_MODERN = begin
```

Comment:
Could this constant _also_ have an initial check `WINDOWS && begin` so that we do not ever execute `ver` on a non-Windows platform?